### ISSUE TITLE
Correcciones pipeline CNN

### ISSUE THREAD

**Issue by josearangos** [josearangos/Voice_Recognition#44]
![image](https://user-images.githubusercontent.com/13352055/76349290-b60cbc00-62d7-11ea-9e06-346e24a20927.png)

**Comment by deiry** [comment]
Redondear las cifras de error
Saber si son resultados de train, validation o test
Corregir sobreajusta de las CNN con dropout y BatchNormalization
Revisar perdida con sparse categorical
Usar tensorboard
Calcular los resultados a mano con una imágen, con las propabilidad  y la crossentropy
Verificar división de datos
